samples/Calling/src/app/views/CallScreen.tsx

```diff
@@ -52,6 +52,10 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
       adapter.onStateChange((state: CallAdapterState) => {
         const pageTitle = convertPageStateToString(state);
         document.title = `${pageTitle} - ${webAppTitle}`;
+
+        if (state.call?.state === 'Connected') {
+          console.log(`CallId: ${state.call.id}`);
```

Comment:
note: this will log on every adapter state change after the call has connected - is there a nice way to only do this when the call connects?

Reply:
Better event to use: calladapter.on('callIdChanged'

Reply:
`callIdChanged` is not what you think it is. You are the third person on our team confused about it this morning.

We really need to rethink this event. It's just piping up the `idChanged` event from the headless SDK `call`. Presumably, that fires if the ID of an *ongoing call* changes. I have no idea why that can happen, but definitely not the case we're looking for here -- we just want to know when we joined a call.

I'd recommend: Remove the `callIdChanged` event from adapter interface (can add back later) because it's confusing.
Consider adding a `callJoined` event, or `pageChanged` event, as Alex suggested.

Reply:
As for logging each time -- you should probably set a `useRef` here, and `console.log` in a `useEffect` bloc whenever that ref changes.